### PR TITLE
General updates on layer5-ng

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR fixes few paths and creates redirects for projects

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/projects/distributed-performance-management-of-service-meshes/index.mdx [hunk 1]
```diff
@@ -2,6 +2,9 @@
 title:  "Distributed Performance Management of Service Meshes"
 thumbnail: ./distributed-performance_green.svg
 published: true
+redirect_from: 
+ - /distributed-performance
+ - /projects/distributed-performance
 ---
```
Comment (leecalcote) [inline src/collections/projects/distributed-performance-management-of-service-meshes/index.mdx:8]: What will the permalink be? 

/projects/distributed-performance-management?

The of-service-meshes portion my too long of a URL.

**Review by leecalcote — APPROVED** [review]
Looking good.

File: src/collections/projects/distributed-performance-management-of-service-meshes/index.mdx (hunk 1 above)
Comment (Jashpatel1) [reply src/collections/projects/distributed-performance-management-of-service-meshes/index.mdx:8]: We did discuss about including a permalink in the frontmatter. Views @Tanuj22 @Nikhil-Ladha on having a pernalink field.

Comment (Tanuj22) [reply src/collections/projects/distributed-performance-management-of-service-meshes/index.mdx:8]: Should like a good inclusion I think

**Pull request merged**